src/amcrest/special.py

```diff
@@ -153,3 +155,62 @@ def mjpg_stream(self, channelno=None, typeno=None, path_file=None):
                 raise CommError(error)
 
         return ret.raw
+
+    def get_event_stream(self, eventcode='VideoMotion', callback=None,
+                         timeout=(TIMEOUT_HTTP_PROTOCOL, 3600*24*365)):
```

Comment:
You can use `None` as the read timeout (i.e., second value) in the timeout tuple, which means pretty much it won't time out. I tried it and the response contained: `Expires: Thu, 01 Dec 2099 16:00:00 GMT` (so, effectively, never.)